site/docs/reference/Connectors/capture-connectors/apache-kafka.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 2
+---
+# Apache Kafka
+
+This connector captures streaming data from Apache Kafka topics.
+
+`ghcr.io/estuary/source-kafka:dev` provides the latest connector image when using the Flow GitOps environment. You can also follow the link in your browser to see past image versions.
+
+## Prerequisites
+
+* A Kafka cluster with:
+  * [bootstrap.servers](https://kafka.apache.org/documentation/#producerconfigs_bootstrap.servers) configured so that clients may connect via the desired host and port
+  * An authentication mechanism of choice set up (highly recommended for production environments)
+  * Connection security enabled with TLS (highly recommended for production environments)
+
+### Authentication and connection security
+
+Neither authentication nor connection security are enabled by default in your Kafka cluster, but both are important considerations.
+Similarly, Flow's Kafka connectors do not strictly require authentication or connection security mechanisms.
+You may choose to omit them for local development and testing; however, both are strongly encouraged for production environments.
+
+A wide [variety of authentication methods](https://kafka.apache.org/documentation/#security_overview) is available in Kafka clusters.
+All SASL methods are permissable for clusters used with Flow, so long as connection details are provided.
```

Comment:
I'm not 100% sure that we support all of them. I've setup/tested SCRAM-256, SCRAM-512, and PLAIN. I vaguely remember the other SASL mechanisms being more complicated than just username+password.